triage/nw_prj_neuron_track_hours/bonita_resolver.py

```
@@ -0,0 +1,356 @@
+    worked_ws = _find_sheet(wb, "Worked Projects", label)
+    if worked_ws is None:
+        resolution.warnings.append(
+            f"worked_projects_missing:{label}:falling_back_to_default_and_assignments"
+        )
+    worked = _worked_project_lookup(worked_ws)
+
+    assignments: Dict[Tuple[date, str], str] = {}
+    try:
+        from triage.roster_parser import _find_assignments_sheet, _load_assignments
+        assignments = _load_assignments(_find_assignments_sheet(wb, label))
+    except Exception as exc:  # pragma: no cover - assignments are optional
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)